On The Map/StudentInformation.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Student.swift
+//  On The Map
+//
+//  Created by Avinash Mudivedu on 4/13/16.
+//  Copyright © 2016 Avinash Mudivedu. All rights reserved.
+//
+
+import Foundation
+
+struct StudentInformation {
+    
+    var objectId:String?
+    let uniqueKey: String
+    let firstName: String
+    let lastName: String
+    var mapString: String?
+    var mediaURL: String?
+    var latitude: Double?
+    var longtitude: Double?
+    
+    init(uniqueKey: String, firstName: String, lastName: String) {
```

Comment:
Add a comment about why this initializer is useful. I figured you only need the one dictionary initializer to build your data model from the JSON request, but this is used somewhere else, and it seems incomplete / hacky / buggy.